### ISSUE TITLE
Consider adding payment method support on <podcast:funding>

### ISSUE THREAD

**Issue by mattbasta** [Podcastindex-org/podcast-namespace#67]
It should be possible to note which payment method types are supported for a given method of funding. Users are strongly motivated to use payment methods which are well-supported in their area. PayPal availability, for instance, may drive South American users to make contributions. Credit Card support may drive US and UK users to make contributions. Some countries may have very high non-card payment method adoption (iDEAL, SEPA, FPX, WeChat Pay, etc.).

**Comment by keunes** [comment]
While it could be useful, I'm not sure that's such a good idea. There are hundreds of payment methods out there and they likely come and go as well. I wouldn't trust podcasters to correctly set the payment methods their payment provider supports. Such data would have to be manually updated by creators and would quickly get out of date (thus unreliable).

**Comment by tomrossi7** [comment]
> There are hundreds of payment methods out there and they likely come and go as well.

Yeah, this is what makes me hesitant to put limitations around this tag. I think the *only* thing we can specify with any certainty that will work is URL.

**Comment by daveajones** [comment]
Getting too specific on this tag seems risky for an additional reason - that it brings attention to this in the eyes of the app stores.  Right now it's largely ignored.  But, the more specific the payment data on this tag gets, the more likely they are to see it as an unauthorized in-app payment.

**Comment by mattbasta** [comment]
I'm not sure I'd be concerned about app store fallout here, there's a lot of indirection here that Apple would be hard pressed to argue past. In fact, it could have the opposite effect if Apple Pay and friends were formally recognized.

I don't have especially strong feelings here, but experience has proven that payment methods drive adoption around features that move money.

**Comment by daveajones** [comment]
Do you have a more formal idea of how that would look that you could post?  I'm interested in what you have in mind.

**Comment by mattbasta** [comment]
I think the design of the tag should be expanded to be a composite tag, which would allow it to be extended in the future more easily, and enable folks (e.g., RadioPublic) who already extend this functionality to write their own extensions _on top_ of `<podcast:funding>`:

```
<podcast:funding href="...url...">
  <podcast:fundingCta>Donate</podcast:fundingCta>
  <podcast:fundingPaymentMethod type="paypal" />
  <podcast:fundingPaymentMethod type="card-payments" />
</podcast:funding>
```

Perhaps something like `podcast:fundingPaymentMethod` would be more useful outside of this spec and be its own sub-spec, but having the root tag be a wrapper that accepts other tags would enable this.

**Comment by daveajones** [comment]
I like this.  We can look at doing this in phase 2 in a way that doesn't break simplicity for those that just want the dead simple href/slogan version.

**Comment by Marzal** [comment]
Hi, as per https://github.com/Podcastindex-org/podcast-namespace/discussions/749 and https://github.com/Podcastindex-org/podcast-namespace/discussions/695 we are closing old issues.

Please open a new [Discussions](https://github.com/Podcastindex-org/podcast-namespace/discussions) if you think that the matter it's worth pursuing.
